release/RELEASE_GUIDELINES.md

```diff
@@ -5,6 +5,8 @@ We follow semver for versioning. Given a version number MAJOR.MINOR.PATCH, incre
 - MINOR version when you add functionality in a backwards-compatible manner
 - PATCH version when you make backwards-compatible bug fixes
 
+For LTS releases, append the "-LTS" suffix to the release version
```

Comment:
Just for clarity, does this mean that this "-LTS" prefix should be applied everywhere, i.e. in  `CHANGELOG` versions and Github release titles?

Reply:
So for LTS releases I can just run `VERSION=x.x.x-LTS node release/release.js` . Has the release script already be updated to be able to handle LTS releases>

Reply:
I don't think the release script supports LTS releases. I think this change should go on the manual release process on confluence